src/components/Integrations/Shopify/Shopify.js

```diff
@@ -90,16 +93,43 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  const getSubscribersAmountFromAPI = async (listId, apikey) => {
```

Comment:
Maybe, I would also put this code in `getShopifyData`.

Reply:
Same here arguable.. 😄